CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Make Inventory search and browse query boxes expandable. Refs UIIN-2493.
 * Added support for `containsAny` match option in Advanced search. Refs UIIN-2486.
 * Inventory search/browse: Do not retain checkbox selections when toggling search segment. Refs UIIN-2477.
+* Switch from `=` to `==` operator when querying for `holdings-storage/holdings` by hrid. Fixes UIIN-2658.
```

Comment:
If this should be released as a Poppy bug fix then move it to v10.0.3, please

Reply:
@mariia-aloshyna I just moved it under `v11.0.3`

Reply:
Hi @mkuklis looks like it was removed from `v11.0.0` but wasn't moved to `v10.0.3`